WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderDiscoverViewModel.kt

```diff
@@ -258,12 +258,13 @@ class ReaderDiscoverViewModel @Inject constructor(
                     )
                 }
             }
+            trackContentPresentedOnce()
```

Comment:
❓ Do we intentionally track even errors? AFAIU it won't help us answer `The % of users that leave the Reader screen before the content appears.` -> assuming error isn't considered as content.

Reply:
I tracked errors intentionally for two reasons:
1) iOS was tracking errors as well.
2) I was trying to track all state changes, except for `LoadingState`. So, both, `ContentUiState` and `EmptyUiState`.

Reply:
@malinajirka we track that because even if the API fails, the user is still there. Perhaps the name of the event is kinda misleading, if you both think that we should change the name, let's do it. :)